src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -1678,7 +1768,7 @@ details.rustdoc-toggle[open] > summary.hideme::after {
 		padding: 0;
 	}
 
-	.sidebar .logo-container {
+	.rustdoc:not(.source) .sidebar .logo-container {
```

Comment:
Seeing a bunch of `not()` selectors usually hints that we're missing some positive class that would be useful. For instance it looks like it would be useful to add a `doc` class to the body tag whenever we're not in source view. But I don't feel that's required as part of this PR.